monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -215,7 +215,7 @@ groups:
             rate of the past 48 hours.
 
       - alert: MTU Mismatch
-        expr: node_network_mtu_bytes{device!="lo"} != on(device) group_left() (quantile(0.5, node_network_mtu_bytes{device!="lo"}) by (device))
+        expr: node_network_mtu_bytes{device!="lo"} != on() group_left() (quantile(0.5, node_network_mtu_bytes{device!="lo"}))
```

Comment:
Is this change supposed to be here?

Reply:
Yes, while writing the unit test for this alert, I found out that this change should be done in order to get the correct alert. I have created a separate PR for this change (https://github.com/ceph/ceph/pull/39462). But in order to run these tests successfully, I did this change in this PR as well.